metadata-models/src/main/pegasus/com/linkedin/metadata/entity/AzkabanJobEntity.pdl

```diff
@@ -0,0 +1,19 @@
+namespace com.linkedin.metadata.entity
+
+import com.linkedin.common.AzkabanJobUrn
+
+/**
+ * Data model for a Azkaban job entity
+ */
+record AzkabanJobEntity includes AzkabanFlowBaseEntity {
```

Comment:
Why do we have flow metadata as properties for Azkaban job nodes? This is kind of unnecessarily replicated information. Job nodes should only have job specific metadata and flow nodes similarly. There will already be edges between these nodes, anyways.

Reply:
makes sense. removed it from job entity